bin/main/docker/mysql/init/ddl.sql

```diff
@@ -0,0 +1,128 @@
+SET CHARSET UTF8;
+SET FOREIGN_KEY_CHECKS = 0;
+drop table if exists users cascade;
+CREATE TABLE users (
+    user_id BIGINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    age INT(3),
+    tel VARCHAR(255),
+    address VARCHAR(255),
+    email VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    delete_flg BOOLEAN NOT NULL DEFAULT 0
+);
+
+drop table if exists categories cascade;
+CREATE TABLE categories (
+    category_id BIGINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    delete_flg BOOLEAN NOT NULL DEFAULT 0
+);
+
+drop table if exists products cascade;
+CREATE TABLE products (
+    product_id BIGINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    category_id BIGINT NOT NULL,
+    sale_user_id BIGINT NOT NULL,
+    buy_user_id BIGINT,
+    name VARCHAR(255) NOT NULL,
+    price INTEGER NOT NULL,
```

Comment:
I
```suggestion
    price INT NOT NULL,
```
上記で良い気がしました！
usersのageがINT、多くのid系がBIGINTなので！